source/fundamentals/serialization.txt

```diff
@@ -140,6 +140,54 @@ Implementing the `IBsonArraySerializer
 interface enables the driver to access serialization information for individual
 items in an array.
 
+.. _csharp-convention-packs:
+
+Convention Packs
+----------------
+
+Convention packs allow you to define and apply
+**conventions** to your classes and their members during serialization.
```

Comment:
"during serialization"

Technically conventions are applied *before* serialization while auto configuring class maps.

Reply:
Updated the wording here. It was meant to convey that the conventions you specify are used/followed in the serialization process